### ISSUE TITLE
Thumbnail nav - Text indication of number of canvases available

### ISSUE THREAD

**Issue by camillevilla** [ProjectMirador/mirador#2109]
This one of several tickets aimed at making the thumbnail nav more accessible. Currently, a user with a screenreader cannot determine how many canvases are available to them. We lazy-load the thumbnails to improve performance, so HTML representations of them are not immediately available. 

One possible option we discussed is providing some `aria` attribute indication of how many canvases are available in the markup. 

## Out of scope
A visual indicator of how many canvases are available. Let's loop back with the design team to flesh this out.

**Comment by jkeck** [comment]
After further discussion, this is not currently actionable.

**Comment by camillevilla** [comment]
Talked this over with @ggeisler and we're closing this in favor of #2114, which announces the total number of canvases via the canvas nav controls. Canvas nav controls will always be accessible to users, whereas the thumbnail strip may be hidden by default by an implementer.
